src/main/java/com/example/core/services/CoreService.java

```diff
@@ -31,30 +33,37 @@ public CoreService(ParserService ps) {
         this.ps = ps;
     }
 
-    public void core(dataDTO data){
+    public void core(dataDTO data) {
         hs.contentLength = data.getContent().length();
         Data _input = new Data(data);
-        hs.updateStatus(_input, 0, 0);
-        List<Analysis> results = new ArrayList<>();
-        hs.updateStatus(_input, 1, 1);
```

Comment:
I actually removed this one. The server does not accept the same step number 2x in a row